scripts/cc-cli/src/test/integration-tests/registerAddress.test.ts

```diff
@@ -39,35 +40,52 @@ describe("register-address", () => {
     await ccApi.api.disconnect();
   });
 
-  test("e2e", async () => {
-    const { api } = ccApi;
+  it.each([
+    ["using ethereum private key", false],
+    ["Using an ethereum mnemonic", true],
+  ])(
+    "should be able to register address",
```

Comment:
```suggestion
    "should be able to register address %s",
```

otherwise doesn't print the value of the test parameter.